apps/Unciv/description

```diff
@@ -0,0 +1,3 @@
+An open-source, mod-friendly Android+Desktop remake of Civ V
```

Comment:
```suggestion
An open-source, mod-friendly Android+Desktop remake of Civ V

A reimplementation of the most famous civilization-building game ever—fast,
small, no ads, free forever!

Build your civilization, research technologies, expand your cities and
defeat your foes!
```

Reply:
Did you make it up yourself 🤣! Anyways, It lookes very appealing!